wazo_sysconfd/plugins/systemd_services/exceptions.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+class InvalidActionException(ValueError):
+    def __init__(self, service_name, action):
+        super(InvalidActionException, self).__init__(self)
```

Comment:
Python 3 allows the use of the shorter version `super().__init__()`

Reply:
Done